lib/profile/WP_Auth0_Profile_Change_Password.php

```diff
@@ -44,14 +52,17 @@ public function init() {
 	public function validate_new_password( $errors, $user ) {
 
 		// Exit if we're not changing the password.
-		if ( empty( $_POST['pass1'] ) ) {
+		// The pass1 key is for core WP, password_1 is WooCommerce.
+		if ( empty( $_POST['pass1'] ) && empty( $_POST['password_1'] ) ) {
 			return false;
 		}
-		$new_password = $_POST['pass1'];
+
+		$field_name   = ! empty( $_POST['pass1'] ) ? 'pass1' : 'password_1';
```

Comment:
- please rename this to something more specific like `$psw_field_name` 
- what if `$_POST['password_1']` is empty?

Reply:
> Please rename this to something more specific like $psw_field_name

We're in a validate password method, that's a bit overkill. 

> what if $_POST['password_1'] is empty?

See lines above, if they're both empty, the method ends early.